### PR TITLE
Added header.html

### PR DESCRIPTION
**Pull request by SwatiRaoDataEnthusiast**
Created header.html for the second design

### CODE REVIEW

**Comment by proful** [comment]
create a separate header.css file

screenshot?

**Comment by SwatiRaoDataEnthusiast** [comment]
image

**Comment by arthurbuhl** [comment]
Here is a small demo: https://jsfiddle.net/s4q2d8cr/

**Comment by SwatiRaoDataEnthusiast** [comment]
I have used inline css

**Comment by SwatiRaoDataEnthusiast** [comment]
Thanks @arthurbuhl

File: playground/header.html
Comment (sarathkumar6) [inline playground/header.html:5]: @SwatiRaoDataEnthusiast We would want to keep the styles and html content separate; to avoid maintainability issues as the file will start to grow moving forward.

- [ ] Move styles into its own file and use <link> tag to include that html

File: playground/header.html
Comment (sarathkumar6) [inline playground/header.html:6]: Minor: Formatting inconsistency

File: playground/header.html
Comment (sarathkumar6) [inline playground/header.html:20]: Please use flexbox to keep things responsive, although float works but not on all the devices the its consistent

- [ ] Use flexbox

**Review by sarathkumar6 — COMMENTED** [review]
@SwatiRaoDataEnthusiast Good first step. Appreciate it.

File: playground/header.html
Comment (SwatiRaoDataEnthusiast) [reply playground/header.html:20]: I have to learn about flexbox .So it will take some time

Comment (sarathkumar6) [reply playground/header.html:20]: No problem @SwatiRaoDataEnthusiast Take your time.
Here is a useful link to learn about flexbox
https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Comment (proful) [reply playground/header.html:20]: I think for first pr after she split it up we should approve.

Flexbox can be next PR. 

I agree to encourage to use Flexbox.

**Comment by SwatiRaoDataEnthusiast** [comment]
I have used flexbox and separated html and css

**Comment by SwatiRaoDataEnthusiast** [comment]
image

**Review by sarathkumar6 — APPROVED** [review]

**Comment by arthurbuhl** [comment]
> 
> 
> Here is a small demo: [jsfiddle.net/s4q2d8cr](https://jsfiddle.net/s4q2d8cr/)

Here is the updated demo using flexbox: https://jsfiddle.net/9phfy8b6/

**Review by UnevenCoder — APPROVED** [review]
Good to go

**Review by tusharkandpal — COMMENTED** [review]
@SwatiRaoDataEnthusiast please change `styles.css` to `header.css` as per naming conventions

Also please do remember to change it inside your html file

**Review by tusharkandpal — APPROVED** [review]
All good now 👍🏻

File: playground/header.css
Comment (sarathkumar6) [inline playground/header.css:3]: Minor: Formatting could be improved

**Review by sarathkumar6 — APPROVED** [review]

**Pull request merged**
